src/components/Sidebar.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+    import History from '../components/History.svelte';
+</script>
+
+<div class="h-[calc(100%-70px)] float-right flex flex-col items-start pl-6 py-6 gap-6 bg-[rgb(27,30,35)]/50">
+    <History />
+</div>
```

Comment:
I'm not sure this is worth being a component and having a separate file. Its just a stylized div that probably won't be re-used since there is only one page. I would just move the div to the page and import History to that.